libcontainer/system/proc.go

```diff
@@ -78,7 +78,7 @@ func GetProcessStartTime(pid int) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return fmt.Sprintf("%d", stat.StartTime), nil
+	return strconv.FormatUint(stat.StartTime, 10), nil
```

Comment:
Wonder if `GetProcessStartTime` has any users left? I looked around, can't find any. Maybe we can just remove it.

Reply:
Can and should be addressed separately of course.